### DISCUSSION TITLE
10ten for android?

### DISCUSSION THREAD

**Discussion by PjPajkis** [birchill/10ten-ja-reader#889] [Q&A]
Some time ago I was using (not sure if rikaichamp) on firefox as android extension on mobile phone but non of these extensions seems not to be working anymore. Is there any chance that 10ten would be available for androids?

**Comment by birtles** [comment]
Hi! You can't use 10ten on the latest release versions of Firefox for Android because Mozilla only allows extensions in the Recommended add-ons program to be installed there (and language-specific tools will never qualify for that program).

You can use 10ten on Firefox Nightly for Android, however, and also on Kiwi browser. You can read instructions in this Twitter thread: https://twitter.com/10tenstudy/status/1471648846316916739
